### PR TITLE
[CoW Protocol] Add Parsed Order Rewards

### PR DESCRIPTION
**Pull request by bh2smith**
Similar to #2302, we create a view which parses the unstructured content from `cowswap.raw_order_rewards` in to a table with unpacked `data` field.

PoC Query - [here](https://dune.com/queries/1752782)

cc @josojo and @gentrexha for internal review.

### CODE REVIEW

File: models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+                                    \'["bh2smith"]\') }}'
+)}}
+
+-- PoC Query here - https://dune.com/queries/1752782
+select
+    tx_hash,
+    data.amount as cow_reward,
+    cast(data.surplus_fee as double) as surplus_fee,
+    order_uid
+from cowswap.raw_order_rewards
```
Comment (bh2smith) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql:14]: This is not using a ref call but CI does not complain... Not sure yet how to reference community sources...

Comment (bh2smith) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql:14]: According to this:

https://github.com/duneanalytics/spellbook/blob/ad2b91fef757f12a885fd2b71df293319e047f68/models/labels/flashbots/labels_flashbots_ethereum.sql#L15-L15

I should use source(`cowswap`, 'raw_order_rewards')

Comment (bh2smith) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql:14]: Fixed

**Review by jeff-dude — APPROVED** [review]
this is the right approach after reading in community data elsewhere 👍

**Pull request merged**